setup.py

```diff
@@ -18,7 +18,7 @@
     author="Yoshiki Vazquez-Baeza",
     author_email="yoshiki@ucsd.edu",
     description="Display ordination plots",
-    license="bsd",
+    license='BSD-3-Clause',
```

Comment:
conda-forge doesn't have a standard to my knowledge, it was just an opinion of who we were working with. I would rather match a standards body.